.gitlab-ci-check-apidocs.yml

```diff
@@ -32,3 +32,54 @@ test:apidocs:
     - swagger-cli validate docs/*.yml
     # Verify that the Swagger docs follow the autodeployment requirements
     - python3 verify_docs.py `find docs -name "*.yml"`
+
+trigger:apidocs:rebuild-mender-api-docs:
+  stage: .post
+  only:
+    changes:
+      - docs/*
+    refs:
+      - /^(master|staging|[0-9]+\.[0-9]+\.x)$/
+  image: alpine
+  before_script:
+    - apk add git python3 py3-pip
+    - pip3 install pyyaml
+    - git clone https://github.com/mendersoftware/integration.git mender-integration
+    - alias release_tool=$(realpath mender-integration/extra/release_tool.py)
+    - apk add --no-cache curl
+  script:
+    # If the repo is not recognized, ignore
+    - if ! echo $(release_tool --list git --all) | grep $CI_PROJECT_NAME; then
+    -  echo "Repository $CI_PROJECT_NAME not found in release_tool. Exiting"
+    -  exit 0
+    - fi
```

Comment:
As we don't expect any non-releasable repo to have API specs, I suggest we remove this now, and add it if we ever have the need.

This has a risk of potentially hide an error in the release_tool or something.

Reply:
Ack